armi/reactor/components/componentParameters.py

```diff
@@ -16,6 +16,7 @@
 from armi.reactor import parameters
 from armi.reactor.parameters import ParamLocation
 from armi.utils import units
+from armi.reactor.parameters.parameterDefinitions import isNumpyArray
```

Comment:
We could alphabetize these imports. Since we're here.

Reply:
https://github.com/terrapower/armi/pull/1954/commits/e285bd3f48c8b0a6ad793c8b97391b407078ddec